index.html

```diff
@@ -4,7 +4,7 @@
   <meta charset="UTF-8">
```

Comment:
### Kindly add a report in your PR description
- The report includes a check about page titles.
- The report includes a check about image text alternatives (`<img alt="alt text ">`).
- The report includes a check about text headings.
- The report includes a check about ** color contrast**.
- The report includes a check about resizing.
- The report includes a check about interaction.
- The report includes a check about moving content.
- The report includes a check about multimedia.
- The report includes a check on the basic structure of the page.

### In the result section of the report should include a statement pertained to every point checked:

If everything was ok, Please make a statement about that (e.g. "Page titles - no issues found").
If issues were found, Please provide a link to the corresponding line of code in the Pull Request changes